network-api/networkapi/donate_banner/models.py

```diff
@@ -51,11 +51,42 @@ class DonateBanner(TranslatableMixin, PreviewableMixin, models.Model):
             "or a valid query string starting with ? (Ex: ?form=donate)."
         ),
     )
-    background_image = models.ForeignKey(
+    foreground_image = models.ForeignKey(
         "wagtailimages.Image",
         models.PROTECT,
         related_name="+",
     )
+    bg_image = models.ForeignKey(
+        "wagtailimages.Image",
+        models.PROTECT,
+        related_name="+",
+        null=True,
+        blank=True,
+    )
+
+    TAILWIND_COLORS = [
+        ("bg-gray-60", "Gray"),
+        ("bg-red-60", "Red"),
+        ("bg-blue-60", "Blue"),
+        ("bg-cyan-60", "Cyan"),
+        ("bg-green-60", "Green"),
+        ("bg-yellow-60", "Yellow"),
+        ("bg-purple-60", "Purple"),
+        ("bg-pink-60", "Pink"),
+    ]
+
+    bg_color = models.CharField(
+        max_length=20, choices=TAILWIND_COLORS, default="bg-blue-60", help_text="Background color for the banner"
+    )
+
+    TEXT_COLORS = [
+        ("text-white", "White"),
+        ("text-black", "Black"),
+    ]
+
+    text_color = models.CharField(
+        max_length=20, choices=TEXT_COLORS, default="text-white", help_text="Text color for the banner"
+    )
```

Comment:
Would it be possible to include `tw-` in these class names? It would make the classes easier to search for in the future!

Reply:
Sure. I went without the prefix in this approach contemplating the bootstrap removal would eventually make it go away, and would be portable. But I think you're right, in order to track correctly instances of tailwind classes at the moment we should be consistent in the way we express those in the codebase. Will update.